lua/cmake-tools/session.lua

```diff
@@ -1,6 +1,22 @@
 local osys = require("cmake-tools.osys")
 local utils = require("cmake-tools.utils")
 
+local function findFileInDirOrParent(dir, fname)
```

Comment:
Change it to `find_file_in_dir_or_parent`.

Reply:
And I think it's better to just search `.cmake-tools.lua` from current working directory. Current approach may lead to recursion bug. Sometimes we better restrict our power.

Reply:
> Change it to `find_file_in_dir_or_parent`.

done

> And I think it's better to just search `.cmake-tools.lua` from current working directory. Current approach may lead to recursion bug. Sometimes we better restrict our power.

I find it rather useful, since it allows me to set good presets for all my repos in the parent folder and fine tune them in the individual ones if needed...